README.md

```
@@ -1,3 +1,9 @@
+[![CodeQL](https://github.com/ETERNA-earkiv/ETERNA/actions/workflows/github-code-scanning/codeql/badge.svg)](https://github.com/ETERNA-earkiv/ETERNA/actions/workflows/github-code-scanning/codeql)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)